### PR TITLE
Adding iOS docs for RNPM/CocoaPods installation

### PR DESCRIPTION
**Pull request by lostintangent**
This PR simply introduces instructions for how to install the CodePush plugin into iOS apps using either RNPM or CocoaPods. We can add Android/RNPM instructions in a separate PR.

### CODE REVIEW

**Comment by dtivel** [comment]
Looks good.

**Comment by shishirx34** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -80,16 +83,54 @@ If you want to see how other projects have integrated with CodePush, you can che
...
+
+1. [**RNPM**](#plugin-installation-ios---rnpm) - [React Native Package Manager (RNPM)](https://github.com/rnpm/rnpm) is an awesome tool that provides the simplest installation experience possible for React Native plugins. If you're already using it, or you want to use it, then we recommend this approach. 
+
+2. [**CocoaPods**](#plugin-installation-ios---cocoapods) - If you're building a native iOS app that is embedding React Native into it, or you simply prefer using [CocoaPods](https://cocoapods.org), then we recommend using the Podspec file that we ship as part of our plugin.
+
+3. [**"Manual"**](#plugin-installation-ios---manual) - If you don't want to depend on any additional tools or are fine with a few extra installation steps (it's a one-time thing), then go with this approach.
+
+#### Plugin Installation (iOS - RNPM)
+
+1. Run `rnpm link react-native-code-push`
```
Comment (geof90) [inline README.md]: Should we include a step to `npm install -g rnpm` if they have not already done it? Just to be super idiot proof.

File: README.md [hunk 2]
```diff
@@ -80,16 +83,56 @@ If you want to see how other projects have integrated with CodePush, you can che
...
+
+    *Note: If you didn't already have RNPM installed, you can do so by running `npm i -g rnpm` and the executing the above command once `rnpm` is in your path.*
```
Comment (geof90) [inline README.md]: and then executing

**Comment by geof90** [comment]
LGTM!

**Pull request merged**